nsv13/code/modules/turbolift/lazylift.dm

```diff
@@ -344,17 +350,15 @@ That's it, ok bye!
 	for(var/obj/machinery/lazylift/L in decks) //First pass: Find the target
 		if(L.deck == targetDeck)
 			target = L
-			if(target == src)
-				for(var/turf/T in platform)
-					for(var/atom/movable/AM in T)
-						if(isturf(AM))
-							continue
-						if(isliving(AM))
-							var/mob/living/karmics_victim = AM //SQUISH
-							karmics_victim.gib() //Elevator suicides are...a thing I guess!
-						else
-							AM.ex_act(4) //Crush
-			break
+			if(target != src)
+				break
```

Comment:
I think the original breaks regardless of whether target == src

Reply:
Oh good point. This code makes even LESS sense now xd